src/lib/output/themes/default/partials/member.declaration.tsx

```diff
@@ -29,12 +29,44 @@ export function memberDeclaration(context: DefaultThemeRenderContext, props: Dec
 
     const visitor = { reflection: renderTypeDeclaration };
 
+    /** Fix for #2717. If type is the same as value the type is omited */
+    function shouldRenderType() {
+        if (props.type && props.type.type === "literal") {
+            const typeObject = props.type.toObject();
```

Comment:
I'd prefer to not call `toObject` here. That is intended for serialization, and this code ought to work with the non-serialized object instance. You might find `props.type.toString()` useful, however.